### PR TITLE
Update SETUP.md

### PR DESCRIPTION
**Pull request by sdavidp**

### CODE REVIEW

**Review by marcinwadon — DISMISSED** [review]

**Comment by marcinwadon** [comment]
@sdavidp please add a label

File: SETUP.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ On Mac OS, _SBT_ provides instructions [here](https://www.scala-sbt.org/1.x/docs
 ### Update your kubeconfig
 
 ```
+aws configure   # requires AWS Access Key(s)
```
Comment (tbekas) [inline SETUP.md:67]: This paragraph intentionally is about the kubernetes configuration only, not about a configuration of other tools listed here. 

In example `sbt` also has to be configured for this flow to run, but it's omitted here and explained in the [other place](https://github.com/Constellation-Labs/tessellation/blob/develop/CONTRIBUTING.md#setting-up-sbt).

If you wish to explain how to configure AWS CLI, please add a separate paragraph about it.

**Review by tbekas — CHANGES_REQUESTED** [review]

File: SETUP.md (hunk 1 above)
Comment (sdavidp) [reply SETUP.md]: I added the AWS section. I don't understand the comment about `sbt` though.

Comment (marcinwadon) [reply SETUP.md]: @sdavidp in order to use skaffold, you need to configure the sbt first (to compile the project). it's ommited in the skaffold documentation as it's specified in other place (how to configure your dev env)

**Comment by sdavidp** [comment]
@tbekas I'm not clear if there's something outstanding I need to address

**Review by marcinwadon — APPROVED** [review]

**Pull request merged**